html-css/Task3/index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <title>Sitename - Description</title>
+        <meta name="description" content="description">
+        <meta name="keywords" content="site, web, sitename">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="css/style.css" rel="stylesheet">
+        <link href="css/page.css" rel="stylesheet">
+        <link href="css/menu.css" rel="stylesheet">
+        <link href="css/banner.css" rel="stylesheet">
+        <link href="css/article.css" rel="stylesheet">
+        <link href="css/info.css" rel="stylesheet">
+        <link href="css/footer.css" rel="stylesheet">
+        <link href="css/subblock.css" rel="stylesheet">
+        <link href="css/header.css" rel="stylesheet">
+        <link href="css/contact.css" rel="stylesheet">
+        <link href="css/content.css" rel="stylesheet">
+        <link href="css/main.css" rel="stylesheet">
+        <link href="css/fonts.css" rel="stylesheet">
+    </head>
+    <body class="page">
+        <header class="page__header header">
+            <span class="header__sitename">Sitename</span>
+            <ul class="menu header__menu">
+                <li class="menu__item "><a href="#">Home</a></li>
+                <li class="menu__item"><a href="#">About</a></li>
+                <li class="menu__item"><a href="#">Services</a></li>
+                <li class="menu__item"><a href="#">Work</a></li>
+                <li class="menu__item"><a href="#">Contact</a></li>                   
+            </ul>       
+        </header>  
+
+        <main class="page__main main">
+            <div class="banner"> 
+                <section class="banner_header">
+                    <span class="banner__title banner__title_center">
+                        I am the Way,  the Truth,  and the Life. 
+                    </span>
+                    <span class="banner__caption banner__caption_bottom">
+                        No one comes to the Father except through me.
+                    </span> 
+                </section>    
+                <div class="banner__image"></div>  
+            </div>  
+
+            <div class="content">
+                <section class="content__header">
+                    <article class="info">
```

Comment:
`info` block is still less than inside content

Reply:
![image](https://user-images.githubusercontent.com/5310247/42683244-7104cfaa-8695-11e8-88d6-ebbbe5c4d012.png)